src/MoBi.Presentation/Tasks/Interaction/InteractionTasksForModule.cs

```diff
@@ -153,13 +153,13 @@ private List<IBuildingBlock> loadBuildingBlock(BuildingBlockType buildingBlockTy
          switch (buildingBlockType)
          {
             case BuildingBlockType.Reaction:
-               items.AddRange(InteractionTask.LoadItems<ReactionBuildingBlock>(filename));
+               items.AddRange(InteractionTask.LoadItems<MoBiReactionBuildingBlock>(filename));
```

Comment:
if they are sent from PKSim, they are not from type MoBiReactionbUildingblock .dOes this still work?

Reply:
Yes, I checked with a PK-Sim simulation.

Reply:
I think this is why/how
```
public MoBiSpatialStructureXmlSerializer() : base(AppConstants.XmlNames.SpatialStructure)
```